accounts/views.py

```diff
@@ -121,6 +122,39 @@ def login_signup(request):
 
     return render(request, 'accounts/login_signup.html')
 
+
+def resend_verification_mail(request):
+    if request.method == 'POST':
+        email = request.POST['email']
+        user = User.objects.filter(email=email)
+        if user.exists() and user[0].is_active:
+            error = "Your account is already activated"
+            return render(request, "accounts/resend_verification_email.html", {'signup_error': error})
+        if not user.exists():
+            error = "Account with the entered email does not exist"
+            return render(request, "accounts/resend_verification_email.html", {'signup_error': error})
+        user = user[0]
```

Comment:
Add a comment above that this part is to send the activation mail (like done in `login_signup` view).